src/components/ArticleById.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getArticleById } from "./Api";
+
+const ArticleById = () => {
+
```

Comment:
I know there is an error handling ticket but dont forget to think about it now, what happens if this request fails?